frontend/src/components/EventPreview.tsx

```diff
@@ -179,6 +179,7 @@ const EventPreview: FC = () => {
             currentPage={currentPage}
             handleMovements={handleMovements}
           />
+          <div className="mt-5" />
```

Comment:
Our tailwind hasn't defined 5. So you should make sure to use 2,4,6,8 even numbers otherwise it won't apply any margin or padding @glamboyosa